components/automate-ui/src/app/modules/infra-proxy/chef-server-details/chef-server-details.component.html

```diff
@@ -30,18 +30,18 @@
         <form [formGroup]="updateServerForm">
           <chef-form-field id="update-name">
             <label>
-              <span class="label">Chef Server Name <span aria-hidden="true">*</span></span>
+              Chef Server Name <span aria-hidden="true">*</span>
               <input chefInput formControlName="name" type="text" autocomplete="off"
                 data-cy="update-chefServer-name">
             </label>
             <chef-error
               *ngIf="(updateServerForm.get('name').hasError('required') || updateServerForm.get('name').hasError('pattern')) && updateServerForm.get('name').dirty">
-              Display Name is required.
+              Name is required.
             </chef-error>
           </chef-form-field>
           <chef-form-field id="update-description">
             <label>
-              <span class="label">Chef Server Description <span aria-hidden="true">*</span></span>
+              Description <span aria-hidden="true">*</span>
```

Comment:
Since this is required, seems like it should have the same error mechanism as other fields (`...hasError('required')...`).
Same for FQDN below.